packages/libretro/ppsspp/package.mk

```diff
@@ -19,7 +19,7 @@
 ################################################################################
 
 PKG_NAME="ppsspp"
-PKG_VERSION="bebc63c"
+PKG_VERSION="abee4eb"
```

Comment:
@Ntemis  What's wrong with https://github.com/libretro/ppsspp/pull/28 ?

Reply:
OMG sorry i was working on aarch64 asm opts for this core and got distracted :)
Fixed my comment and thanks for the pointing out

Reply:
fixed by editing my comment